src/app-layout/__integ__/app-layout-split-panel.test.ts

```diff
@@ -351,4 +363,50 @@ describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)(
       }, '#/light/app-layout/with-full-page-table-and-split-panel')
     );
   });
+
+  test(
+    'forces split panel to the bottom or allows on side when page scrollbar appears',
+    useBrowser(async browser => {
+      const page = new AppLayoutSplitViewPage(browser);
+      const dimensions = {
+        classic: { height: 1400, width: 945 },
+        'visual-refresh': { height: 1200, width: 1020 },
+        'visual-refresh-toolbar': { height: 1400, width: 875 },
+      }[theme];
+      const content = wrapper.findContentRegion();
+
+      // Split panel is forced to the bottom when not enough horizontal space.
+      await page.setWindowSize(dimensions);
+      const params = new URLSearchParams({
+        splitPanelPosition: 'side',
+        visualRefresh: `${theme.startsWith('visual-refresh')}`,
+        appLayoutToolbar: `${theme === 'visual-refresh-toolbar'}`,
+      });
+      await browser.url(`#/light/app-layout/with-split-panel?${params.toString()}`);
+      await page.waitForVisible(content.toSelector());
+      await page.click(content.findContainer('[data-testid="container-1"]').findHeader().findButton().toSelector());
+      await page.click(content.findContainer('[data-testid="container-1"]').findHeader().findButton().toSelector());
```

Comment:
Do we need to click on the same element twice?

Reply:
Yes, to remove both containers. The test-ids are indices so both have index=1 when in the first position.